watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/GLSD_anaerobic_digester/GLSD_anaerobic_digester_ui.py

```diff
@@ -0,0 +1,456 @@
+        is_output=True,
+        output_category="Feed",
+    )
+
+    # Unit model data, anaerobic digester
+    exports.add(
+        obj=fs.AD.recovery_frac_mass_H2O[0],
+        name="Water recovery",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Water recovery [g-H2O treated/g-H2O inlet]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.reaction_conversion[0, "tss_reaction"],
+        name="TSS conversion",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="TSS conversion [g-TSS reacted/g-TSS inlet]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.generation_ratio["tss_reaction", "methane"],
+        name="CH4 conversion ratio",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=4,
+        description="CH4 mass conversion ratio with respect to TSS [g-CH4 produced/g-TSS"
+        " reacted]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.generation_ratio["tss_reaction", "carbon_dioxide"],
+        name="CO2 conversion ratio",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=4,
+        description="CO2 mass conversion ratio with respect to TSS [g-CO2 produced/g-TSS"
+        " reacted]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.generation_ratio["tss_reaction", "nitrogen"],
+        name="N2 conversion ratio",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=4,
+        description="N2 mass conversion ratio with respect to TSS [g-N2 produced/g-TSS"
+        " reacted]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.generation_ratio["tss_reaction", "oxygen"],
+        name="O2 conversion ratio",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=4,
+        description="O2 mass conversion ratio with respect to TSS [g-O2 produced/g-TSS"
+        " reacted]",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.HRT,
+        name="HRT",
+        ui_units=pyunits.hr,
+        display_units="h",
+        rounding=0,
+        description="Hydraulic retention time",
+        is_input=True,
+        input_category="Anaerobic digester",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.AD.energy_electric_flow_vol_inlet,
+        name="Blower specific power",
```

Comment:
I'm unsure about the inclusion of a blower here. Was this in the team's template? If not, I think this might be residual from one of the activated sludge process flowsheets.

Reply:
I don't know how to resolve this, maybe close this PR and start a new one is easiest

Reply:
About the inclusion of a blower though--was this intentional or carryover from another example? I suppose the process could have a blower to transport gas out of the digester.